File: gluecodium/src/test/resources/smoke/duplicate_names/output/android/com/example/smoke/LearnToReadAgain.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class LearnToReadAgain {
    @NonNull
    public com.example.smoke.foo.Alphabet fieldB;
    @NonNull
    public com.example.smoke.bar.Alphabet fieldC;
    public LearnToReadAgain() {
        this.fieldB = com.example.smoke.foo.Alphabet.BETA;
        this.fieldC = com.example.smoke.bar.Alphabet.GIMEL;
    }
    public LearnToReadAgain(@NonNull final com.example.smoke.foo.Alphabet fieldB, @NonNull final com.example.smoke.bar.Alphabet fieldC) {
        this.fieldB = fieldB;
        this.fieldC = fieldC;
    }
}

File: gluecodium/src/test/resources/smoke/duplicate_names/output/android-kotlin/com/example/smoke/LearnToReadAgain.kt
/*

 *
 */

@file:JvmName("LearnToReadAgain")

package com.example.smoke

import com.example.smoke.bar.Alphabet
import com.example.smoke.foo.Alphabet

class LearnToReadAgain {
    @JvmField var fieldB: Alphabet
    @JvmField var fieldC: Alphabet



    constructor() {
        this.fieldB = Alphabet.BETA
        this.fieldC = Alphabet.GIMEL
    }





}

